### PR TITLE
bsp: imx: add bsp for imx8mp-lpddr4-evk

### PR DESCRIPTION
**Pull request by Tim-Anderson**
Signed-off-by: Tim Anderson <tim.anderson@foundries.io>

### CODE REVIEW

File: bsp/imx/imx8mp-lpddr4-evk-standard.scc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+define KMACHINE imx8mpevk
```
Comment (mike-scott) [inline bsp/imx/imx8mp-lpddr4-evk-standard.scc:1]: This needs an update to match new naming.

File: bsp/imx/imx8mp-lpddr4-evk.scc [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+include features/usb-net/usb-net.scc
+include features/wifi/wifi-pci.scc
+include features/pwm/pwm.scc
+include features/crypto/arm64-crypto.scc
+include features/power/cpu-freq.scc
+include features/tee/tee.scc
+
+include cfg/usb-mass-storage.scc
+
+kconf hardware imx8mpevk.cfg
```
Comment (mike-scott) [inline bsp/imx/imx8mp-lpddr4-evk.scc:49]: This needs an update to match the new filename

**Review by mike-scott — CHANGES_REQUESTED** [review]

**Review by angolini — APPROVED** [review]
nothing else to comment, LGTM

**Review by vanmaegima — APPROVED** [review]
LGTM

**Comment by ricardosalveti** [comment]
Can you update the commit subject line to use the new board name instead?

**Review by ricardosalveti — APPROVED** [review]
LGTM other than my subject machine renaming comment.

**Comment by Tim-Anderson** [comment]
done

**Pull request merged**